core/src/avm2/globals/flash/events/Event.as

```diff
@@ -0,0 +1,129 @@
+package flash.events {
+	[Ruffle(InstanceAllocator = "true")]
```

Comment:
Following from the above, we could allow `[Ruffle(InstanceAllocator)]`?

Reply:
I didn't realize that that was supported - thanks!